lib/athena/README.md

```diff
@@ -0,0 +1,30 @@
+# Athena
+
+A simplified, promise-based library for querying [Athena](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Athena.html). This removes the need write custom polling logic.
+
+## Get a client
+
+```
+const athenaClient = new require('@sparkpost/aws').Athena( opts )
```

Comment:
As someone still pretty new to javaescript, is it common to use `new` when we're not calling a class constructor? The function the module exports just returns an object with a property that's a function.

Reply:
good call, there's no need for the `new`. That's just a hold over from how I was thinking of implementing it.